src/filters/to-year.js

```diff
@@ -0,0 +1,4 @@
+export function toYear(value) {
+  let arr = value.split("-");
```

Comment:
Consider using "const" in cases where a variable is not reassigned in the current scope.
Though not a mistake, consistency across the app help to improve the overall code style.